Nodejs/Product/Nodejs/Snippets/1033/Nodejs/cl.snippet

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>cl</Title>
+      <Description>Code Snippet for a Console message</Description>
+      <Shortcut>cl</Shortcut>
+      <SnippetTypes>
+        <SnippetType>Expansion</SnippetType>
+      </SnippetTypes>
+    </Header>
+    <Snippet>
+      <Code Language="Nodejs">
+        <![CDATA[console.log();]]>

```

Comment:
console.log($message$) instead. There's no reason to have an empty console.log()